backend/diagram/class/ModelPackage.puml

```diff
@@ -0,0 +1,88 @@
+@startuml
+
+package model {
+    class Entity {
+        -Long id
+        +Long getId()
+    }
+
+    class UserBuilder {}
```

Comment:
Maybe we could add the lombok builders here with

```
class UserBuilder {
      == @Builder ==
      +params
}
```

Just like what we did for override functions in other packages so that we know what are our lombok builders by looking at the diagram.

Reply:
Updated!